system_strings/sysmes_text.en

```diff
@@ -4258,219 +4331,354 @@ I can't betray her trust.
 -- ノエル先生を振り返る。^決めていた筈だ。^自分に出来る事をしなければ！
-ノエル先生を振り返る。^決めていた筈だ。^自分に出来る事をしなければ！
+// @flowchart_descr
+I look back at Noel.
+ I've already made my decision.
+ I have to do what I can!
 }
 [sha:27c3ae890805c69dc8367ebe69c6187f6092d29b]{
 -- ドームの崩落に巻き込まれた。^目覚めた場所は暗く、狭く、^そして柔らかい……柔らかい？
-ドームの崩落に巻き込まれた。^目覚めた場所は暗く、狭く、^そして柔らかい……柔らかい？
+// @flowchart_descr
+The underground chamber collapses around me.
+ When I wake up, the world around me is cramped, dark, and soft... soft?!
 }
 [sha:3dd7c368545243f353fd970a61dcb5f71ee8e470]{
 -- ようやく地上に戻る事ができた。^沈みつつある日の光さえ眩しい。^そして、先輩の視線は冷たい。
-ようやく地上に戻る事ができた。^沈みつつある日の光さえ眩しい。^そして、先輩の視線は冷たい。
+// 眩しい tweaked a bit to better mirror 冷たい
+// @flowchart_descr
+We finally make it back to the surface.
+ Even the faint rays of the setting sun feel warm.
+ However, Senpai's gaze is frosty.
 }
 [sha:366e5ab1ff215dd7f6a1785216e2887a9134112c]{
 -- 再びマンションへ戻ってくる。^彼女の機嫌を損なわぬように、^起きた事の全てを話そう。
-再びマンションへ戻ってくる。^彼女の機嫌を損なわぬように、^起きた事の全てを話そう。
+// @flowchart_descr
+I make my way back to her apartment again.
+ In order to quell her anger, I'll tell her all about what happened.
 }
 [sha:910c501d7b667621d16b9194dbef7f08b6b1b625]{
 -- ビルの屋上から街を見下ろす。^瓦礫の中に出現した巨大な結界。^先輩は今も戦っている……！
-ビルの屋上から街を見下ろす。^瓦礫の中に出現した巨大な結界。^先輩は今も戦っている……！
+// @flowchart_descr
+I survey the city from atop a building.
+ A massive barrier has appeared amid the rubble.
+ Senpai's down there fighting as we speak!
 }
 [sha:d525f499037e4e8ef50d37a25fcc230f79eaa94f]{
 -- 階段を駆け下りてゆく。^一刻も早く先輩の元へ行き、^ヴローヴの『線』を断つ……！
-階段を駆け下りてゆく。^一刻も早く先輩の元へ行き、^ヴローヴの『線』を断つ……！
+// @flowchart_descr
+I rush down the stairs.
+ I need to get down to where Senpai is as fast as possible, and sever Vlov's 'line'!
 }
 [sha:dea68eda0403d6acc4908d506c0ea33072cf0932]{
 -- 結界の限界が迫る。^だが、中に入る訳にはいかない。^どうすれば先輩を助けられる？
-結界の限界が迫る。^だが、中に入る訳にはいかない。^どうすれば先輩を助けられる？
+// @flowchart_descr
+I've made it to the threshold of the barrier, but I'm not able to get inside.
+ How am I supposed to help Senpai?
 }
 [sha:12e09f95f6019aafd534b7e88882f693db0a4862]{
 -- シエル先輩の姿を探す。^彼女は瓦礫の山に埋もれている。^今すぐに助け出さなければ！
-シエル先輩の姿を探す。^彼女は瓦礫の山に埋もれている。^今すぐに助け出さなければ！
+// @flowchart_descr
+I hunt around trying to spot Ciel-senpai.
+ There she is――buried in a pile of rubble.
+ I need to go help her right away!
 }
 [sha:d775d08587c287dde2ceaa60b560d0e8b598ab34]{
 -- あの攻撃が負担となったのか、^ヴローヴは動けないようだ。^この隙に先輩を助けなければ。
-あの攻撃が負担となったのか、^ヴローヴは動けないようだ。^この隙に先輩を助けなければ。
+// @flowchart_descr
+It doesn't look like Vlov's able to move...
+ That attack must have taken a lot out of him.
+ I need to seize this chance to help Senpai.
 }
 [sha:ab6da1b4c806a36bf86a89e40ef91829cbd9d0ce]{
 -- ヴローヴに仕掛ける二人。^即興のコンビネーションは、^美しい連弾のように。
-ヴローヴに仕掛ける二人。^即興のコンビネーションは、^美しい連弾のように。
+// @flowchart_descr
+The two of us stand off against Vlov.
+ Our impromptu combination is like a beautiful duet.
 }
 [sha:8514b6be1c4d49262552796f10243bc333b87ee5]{
 -- 花は胸に。^聖女の祈りを贖いに。
-花は胸に。^聖女の祈りを贖いに。
+// Quoting 12930.
+// @flowchart_descr
+A flower to my bosom.
+ Redemption by the Prayer of the Holy Maiden!
```

Comment:
Feel like the chant here is a bit weak, still.
Obviously the same as 12930 so change there too.

I prefer the first one, but the second one I guess works too if you prefer that.
>To the bosom, flowers
>As atonement, the prayers of the holy maiden

>Flowers to the bosom.
>Prayers of the holy maiden for atonement.